tests/drivers/pwm/gpio_loopback/testcase.yaml

```diff
@@ -19,3 +19,33 @@ tests:
       - nrf54h20dk/nrf54h20/cpuapp
       - nrf54l15dk/nrf54l15/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
+
+  drivers.pwm.gpio_loopback.fast_p7_0:
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    extra_args: DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_fast_p7_0.overlay"
+
+  drivers.pwm.gpio_loopback.fast_p7_1:
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    extra_args: DTC_OVERLAY_FILE="boards/nrf54h20dk_nrf54h20_cpuapp_fast_p7_1.overlay"
+
+  drivers.pwm.gpio_loopback.fast_p7_6:
+    build_only: true
```

Comment:
Build only as CI setups contain "incompatible" loopback between P7.6-P7.7.
Test relies on GPIOTE while GPIOTE can't be used on P7.